Sources/BraintreeVenmo/BTConfiguration+Venmo.m

```diff
@@ -22,4 +22,8 @@ - (NSString *)venmoEnvironment {
     return [self.json[@"payWithVenmo"][@"environment"] asString];
 }
 
+- (BOOL)isVenmoEnrichedCustomerDataEnabled {
+    return [self.json[@"payWithVenmo"][@"enrichedCustomerDataEnabled"] isTrue] ?: FALSE;
```

Comment:
```suggestion
    return [self.json[@"payWithVenmo"][@"enrichedCustomerDataEnabled"] isTrue] ?: NO;
```

Reply:
I think NO is the standard in Objective-C to use for `BOOL`

Reply:
🤦